### PR TITLE
Add peppermint bark mimosa recipe

### PR DESCRIPTION
**Pull request by yoji-kojio**
Add a new recipe and fix some indent and blank lines.

### CODE REVIEW

File: src/recipes/peppermint-bark-mimosa.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            "ingredient": "candy canes, whole"
+        }
+    ],
+    "directions": [
+        "Pour melted chocolate onto a small plate. Pour crushed candy canes onto a separate plate.",
+        "Dip rims of champagne flutes first in chocolate, then in crushed candy canes to coat.",
+        "Add an ounce of peppermint schnapps to each glass, then top with champagne or prosecco.",
+        "Garnish with full candy canes before serving."
+    ],
+    "image": "peppermint-bark-mimosa.jpg",
```
Comment (kazuhirodk) [inline src/recipes/peppermint-bark-mimosa.json:38]: I think this must be important
```suggestion
    "image": "peppermint-bark-mimosa.jpg",
    "source": "https://www.delish.com/cooking/recipe-ideas/a25128817/peppermint-bark-mimosas-recipe/",
```

File: src/recipes/peppermint-bark-mimosa.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            "ingredient": "champagne or prosecco"
+        },
+        {
+            "quantity": "6",
+            "measure": "units",
+            "ingredient": "candy canes, whole"
+        }
+    ],
+    "directions": [
+        "Pour melted chocolate onto a small plate. Pour crushed candy canes onto a separate plate.",
```
Comment (kazuhirodk) [inline src/recipes/peppermint-bark-mimosa.json:33]:
```suggestion
        "Pour melted chocolate onto a small plate.",
        "Pour crushed candy canes onto a separate plate.",
```

File: src/recipes/peppermint-bark-mimosa.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        "Pour melted chocolate onto a small plate.",
+        "Pour crushed candy canes onto a separate plate.",
+        "Dip rims of champagne flutes first in chocolate, then in crushed candy canes to coat.",
+        "Add an ounce of peppermint schnapps to each glass, then top with champagne or prosecco.",
+        "Garnish with full candy canes before serving."
+    ],
+    "image": "peppermint-bark-mimosa.jpg",
+    "source": "https://www.delish.com/cooking/recipe-ideas/a25128817/peppermint-bark-mimosas-recipe/",
+    "keywords": [
+        "candy cane",
```
Comment (yoji-kojio) [inline src/recipes/peppermint-bark-mimosa.json:42]:
```suggestion
        "candy",
```

**Comment by alfg** [comment]
Thanks!

**Pull request merged**